crates/ruff_linter/src/rules/pylint/rules/repeated_equality_comparison.rs

```diff
@@ -196,6 +197,18 @@ fn is_allowed_value(bool_op: BoolOp, value: &Expr) -> bool {
         return false;
     }
 
+    if left.is_attribute_expr()
+        && left
+            .to_owned()
+            .expect_attribute_expr()
+            .value
+            .name_expr()
+            .is_some_and(|f| f.id == "sys")
+    {
+        let excluder = Regex::new(r"^(platform|version)").unwrap();
+        return !excluder.is_match(left.clone().expect_attribute_expr().attr.as_str());
+    }
```

Comment:
Thanks! Can you try something like:

```rust
    any_over_expr(test, &|expr| {
        semantic.resolve_call_path(expr).is_some_and(|call_path| {
            matches!(call_path.as_slice(), ["sys", "version_info" | "platform"])
        })
    })
```

This is roughly taken from a helper we have called `is_sys_version_block`.